src/frontend/src/binder/expr/function.rs

```diff
@@ -149,6 +153,53 @@ impl Binder {
             return Ok(TableFunction::new(function_type, inputs)?.into());
         }
 
+        /// TODO: add name related logic
+        fn create_udf_context(
+            binder: &mut Binder,
+            args: &[FunctionArg],
+            _catalog: &Arc<FunctionCatalog>,
+        ) {
+            binder.udf_context = args
```

Comment:
nit: prefer returning value rather than modifying binder directly.
```suggestion
        fn create_udf_context(
            args: &[FunctionArg],
            _catalog: &Arc<FunctionCatalog>,
        ) -> Result<HashSet<..>> {
            args
```